src/components/MiniHistogram/MiniHistogram.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div ref="chart" class="mini-histogram">
+    <template v-for="bar in bars" :key="bar.intervalStart.getTime()">
```

Comment:
Why not use the loop index here?

```suggestion
    <template v-for="(bar, i) in bars" :key="i">
```

Reply:
Using the time means that the bar and animate its position and color changes. If the range changes but this bar is still present it'll animate into place because the key is the same. Using the index can cause that to not work if the bars position in the array changes.

Reply:
Nice ok, that makes sense then